packages/fiori/src/NavigationLayout.ts

```diff
@@ -32,8 +33,8 @@ import NavigationLayoutCss from "./generated/themes/NavigationLayout.css.js";
  * ### Responsive Behavior
  *
  * On desktop and tablet devices, the side navigation remains visible and can
```

Comment:
Please consider change:

On desktop and tablet devices, the side navigation is visible by default and can be expanded or collapsed using the `mode` property. On phone devices, the side navigation is hidden by default and can be displayed using the `mode` property.